src/pluto/image.cr

```diff
@@ -16,21 +16,9 @@ abstract class Pluto::Image
     include Operation::GaussianBlur
     include Operation::HorizontalBlur
     include Operation::VerticalBlur
+    include Operation::Crop
   end
 
-  # TODO: Remove.
-  macro forward_to_rgb_image(*methods)
-    {% for method in methods %}
-      @[Deprecated("Use RGBAImage.{{method.id}} instead")]
-      def self.{{method.id}}(*args, **kwargs)
-        RGBAImage.{{method.id}}(*args, **kwargs)
-      end
-    {% end %}
-  end
-
-  # TODO: Remove.
-  forward_to_rgb_image from_ppm, from_jpg
```

Comment:
I'm _guessing_ these were cleaned up when the `from_ppm(String)` methods went away, so I feel safe removing them